components/esp32_camera.rst

```diff
@@ -80,24 +80,24 @@ Image Settings:
 - **resolution** (*Optional*, enum): The resolution the camera will capture images at. Higher
   resolutions require more memory, if there's not enough memory you will see an error during startup.
 
-    - ``160x120`` (QQVGA)
-    - ``176x144`` (QCIF)
-    - ``240x176`` (HQVGA)
-    - ``320x240`` (QVGA)
-    - ``400x296`` (CIF)
-    - ``640x480`` (VGA, default)
-    - ``800x600`` (SVGA)
-    - ``1024x768`` (XGA)
-    - ``1280x1024`` (SXGA)
-    - ``1600x1200`` (UXGA)
-    - ``1920x1080`` (FHD)
-    - ``720x1280`` (Portrait HD)
-    - ``864x1536`` (Portrait 3MP)
-    - ``2048x1536`` (QXGA)
-    - ``2560x1440`` (QHD)
-    - ``2560x1600`` (WQXGA)
-    - ``1080x1920`` (Portrait FHD)
-    - ``2560x1920`` (QSXGA)
+    - ``160x120`` (QQVGA, 4:3)
+    - ``176x144`` (QCIF, 11:9)
+    - ``240x176`` (HQVGA, 15:11)
+    - ``320x240`` (QVGA, 4:3)
+    - ``400x296`` (CIF, 50:57)
```

Comment:
There seems to be something wrong on this line; 400/296 = 1,35; whereas 50/57 = 1,14

Reply:
Whoops, excellent catch.  You are quite right, it should have read 50:37, not 50:57.  Fixed now :)